yjit_codegen.c

```diff
@@ -1359,6 +1359,43 @@ gen_setlocal_wc0(jitstate_t *jit, ctx_t *ctx, codeblock_t *cb)
     return YJIT_KEEP_COMPILING;
 }
 
+static codegen_status_t
+gen_checkkeyword(jitstate_t *jit, ctx_t *ctx, codeblock_t *cb)
```

Comment:
Can you add a comment explaining what `checkkeyword` does? Is it for checking if a keyword argument was passed or not?

Reply:
yes. Done.